coral/src/domain/requests/requests-types.ts

```diff
@@ -1,8 +1,16 @@
 import { components } from "types/api";
 import { ResolveIntersectionTypes } from "types/utils";
 
-type RequestOperationType = components["schemas"]["RequestOperationType"];
-type RequestStatus = components["schemas"]["RequestStatus"];
+// Missing these two as components/schemas is sad
```

Comment:
😢 

(on the bright side: if we have to do it manually, we could refactor those to enums in the future which would be great for the helper functions. OR sync with BE to provide the schema for that, which would be even better)

Reply:
Yes... But we could also use the relevant property from the relevant entity where we use that type 🤔 

Like this, in `SchemaApprovals`, for example:

![Screenshot 2023-03-16 at 13 03 03](https://user-images.githubusercontent.com/20607294/225611064-f703102d-f3e7-43f7-9c24-ec528c718920.png)